src/Microsoft.ML.Core/Data/Schema.cs

```diff
@@ -0,0 +1,217 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
Note: This file will be coming in via this PR:
[1167](https://github.com/dotnet/machinelearning/pull/1167)